omnigibson/controllers/ik_controller.py

```diff
@@ -50,10 +50,12 @@ def __init__(
             (-0.2, -0.2, -0.2, -0.5, -0.5, -0.5),
             (0.2, 0.2, 0.2, 0.5, 0.5, 0.5),
         ),
+        isaac_kp=None,
+        isaac_kd=None,
         pos_kp=None,
         pos_damping_ratio=None,
         vel_kp=None,
-        use_impedances=True,
+        use_impedances=False,
```

Comment:
are we still keeping `use_impedances=True`? I thought we are getting rid of it entirely.

Reply:
I think for now we can leave it ... post 4.5 once we confirm that everything runs stably without impedances, we can remove completely. What do you think?